src/renderer/shaders/skybox_vs.glsl

```diff
@@ -0,0 +1,15 @@
+#version 330 core
+
+layout(location = 0) in vec3 vertexPosition;
+layout(location = 1) in vec2 vertexTexCoord;
+
+uniform mat4 worldViewProjection;
+
+out vec3 texCoord;
+
+void main()
+{
+    texCoord = vertexPosition;
+    vec4 pos = worldViewProjection * vec4(vertexPosition, 1.0);
+    gl_Position = pos.xyww;
```

Comment:
Just curious, what `.xyww` does here? Is it some trick I'm not aware of?

Reply:
Oh, it makes `pos.z` equal to 1 after perspective division, which places vertex on the far plane of clip space, so that the skybox is always rendered behind other objects. But in our situation it's unnecessary, since we render the sky first. I removed it to avoid confusion.